cms/templates/import.html

```diff
@@ -102,8 +102,8 @@ <h3 class="title">
           <ol class="status-progress list-progress">
             <li class="item-progresspoint item-progresspoint-upload is-complete">
               <span class="deco status-visual">
-                <i class="icon fa fa-cog"></i>
-                <i class="icon fa fa-warning"></i>
+                <span class="icon fa fa-cog" aria-hidden="true"></span>
```

Comment:
It looks like this is probably OK-- the text beside the icons says what is going on, and it does change if there is an error.

![image](https://cloud.githubusercontent.com/assets/484484/15831740/02e5f350-2bed-11e6-864f-0edf0cc83714.png)